projects/nasa-feign/src/main/java/com/bobocode/nasafeign/service/NasaService.java

```diff
@@ -0,0 +1,59 @@
+package com.bobocode.nasafeign.service;
+
+import com.bobocode.nasafeign.client.NasaClient;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.StreamSupport;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class NasaService {
```

Comment:
Does this class partially duplicate NasaServiceRecords? What is the purpose of this class?